sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -150,6 +150,7 @@ export const AESTRoutesConst = {
   ASSESSMENT_AWARD_VIEW: Symbol(),
   REQUEST_CHANGE_OFFERINGS: Symbol(),
   APPLICATION_RESTRICTIONS_MANAGEMENT: Symbol(),
+  APPLICATION_STATUS: Symbol(),
```

Comment:
It may be generic. Should it be, for instance, `APPLICATION_STATUS_TRACKER`?

Reply:
Sure, Just kept the title of side nav button as `Application Status` everything else is modified to have `tracker` in the end.